pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/KitRequestShippingMigrator.java

```diff
@@ -14,11 +16,16 @@ public KitRequestShippingMigrator(String index, String realm) {
 
     /**
      * returns the list of KitRequestShippings for the realm mapped to the participant guid
-     * */
+     */
     @Override
     protected Map<String, Object> getDataByRealm() {
-        Map<String, List<KitRequestShipping>> kitRequests = KitRequestShipping.getAllKitRequestsByRealm(realm, null, null, true);
+        Map<String, List<KitRequestShipping>> kitRequests =
+                KitRequestShipping.getAllKitRequestsByRealm(realm, null, null, true);
+        int recordsFromRealm = kitRequests.size();
+        AdditionalKitShippingRetriever.fromRealm(realm)
+                .ifPresent(retriever -> retriever.mergeRecords(kitRequests));
+        log.info("Migrator retrieved {} medical records from realm {}, and {} additional records",
```

Comment:
This is getting the kit requests, not the MR, right?

Reply:
Right. The logging is messed up and I will add that in a follow-on PR.